pkg/image/common/mutator.go

```diff
@@ -76,11 +80,12 @@ func (m *vmiMutator) getSC(scName string) (*storagev1.StorageClass, error) {
 		return nil, err
 	}
 
-	for _, storageClass := range storageClasses {
-		if storageClass.Annotations[util.AnnotationIsDefaultStorageClassName] == "true" &&
-			storageClass.Provisioner == longhorntypes.LonghornDriverName {
-			return storageClass, nil
-		}
+	defaultSC := util.GetDefaultSC(storageClasses)
```

Comment:
Nit: consider moving the `List()` call to get `storageClasses` into `util.GetDefaultSC()` for re-use purposes. Otherwise, other callers (like `PatchDefaultSC()`) will have to repeat the same `List()` call before they can use `util.GetDefaultSC()`. `util.GetDefaultSC()` can be extended to accept a label argument.